merlin-server/constraints-dsl-compiler/src/main.ts

```diff
@@ -73,8 +86,9 @@ async function handleRequest(data: Buffer) {
     );
 
     if (result.isErr()) {
+      const secondLine = JSON.stringify(result.unwrapErr().map(err => toJson(err))) + '\n';
       process.stdout.write('error\n')
-      process.stdout.write(JSON.stringify(result.unwrapErr().map(err => err.toJSON())) + '\n');
+      process.stdout.write(secondLine);
```

Comment:
If an error happened during the unwrapping, it would print `error` and then go to the `panic` mode (in the catch error block below). Now it goes straight to `panic` and it is clearer in the output.